lib/tramway/forms/normalizations.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Tramway
+  module Forms
+    # This is the same `normalizes` feature like in Rails
+    # https://api.rubyonrails.org/v7.1/classes/ActiveRecord/Normalization/ClassMethods.html#method-i-normalizes
+    module Normalizations
+      # :reek:BooleanParameter { enabled: false }
+      def normalizes(*attributes, with:, apply_to_nil: false)
+        attributes.each do |attribute|
+          @normalizations.merge!(attribute => { proc: with, apply_to_nil: })
+        end
+      end
+
+      def normalizations
+        __ancestor_normalizations.merge(@normalizations)
```

Comment:
Why do you need this if `@normalizations` is already initialized with `__ancestor_normalizations`?

Reply:
Yeah, you're right!

I removed the initialization of `@normalizations` with `__ancestor_normalizations`. Now, it's initialized as properties

```ruby
     def inherited(subclass)
        subclass.instance_variable_set(:@properties, [])
        subclass.instance_variable_set(:@normalizations, {})

        super
      end
```

Reply:
for future: it'll be better to set the variable like `@properties` and `@normalizations` one time, as for now they are recursively generated for each `.properties` and `.normalizations` call